### PR TITLE
Release Prep

### PR DESCRIPTION
**Pull request by motm32**
Notice.html file currently not included since the component governance page is currently.

### CODE REVIEW

**Review by hossam-nasr — APPROVED** [review]
:shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 1.14.0 - 2024-03-20
```
Comment (nturinski) [inline CHANGELOG.md:3]: Gotta change the date.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+### Added
+* Support for creating containerized function apps [[3943]](https://github.com/microsoft/vscode-azurefunctions/pull/3943), [[3964]](https://github.com/microsoft/vscode-azurefunctions/pull/3964)
```
Comment (nturinski) [inline CHANGELOG.md:6]: nit: Could you put the issue links as prefixes like the 1.1.3 release?

**Review by nturinski — COMMENTED** [review]
Any idea why the lockfile when from 2 to 3? It also doesn't seem to list all the subdependencies or something-- I noticed it's quite a bit smaller. Not sure if that's something that we need to be concerned about.

File: CHANGELOG.md (hunk 2 above)
Comment (motm32) [reply CHANGELOG.md:6]: Yah I just did it this way since we have been doing it like this for other extensions but I can change it.

**Comment by motm32** [comment]
> Any idea why the lockfile when from 2 to 3? It also doesn't seem to list all the subdependencies or something-- I noticed it's quite a bit smaller. Not sure if that's something that we need to be concerned about.

Hmm not really sure I didn't change anything just ran `npm install` and that was what it gave me.

File: CHANGELOG.md (hunk 2 above)
Comment (nturinski) [reply CHANGELOG.md]: Alex and I are trying to make more PM friendly notes 😋

**Comment by nturinski** [comment]
> > Any idea why the lockfile when from 2 to 3? It also doesn't seem to list all the subdependencies or something-- I noticed it's quite a bit smaller. Not sure if that's something that we need to be concerned about.
> 
> Hmm not really sure I didn't change anything just ran `npm install` and that was what it gave me.

What nvm/node version do you have?

**Comment by motm32** [comment]
> > > Any idea why the lockfile when from 2 to 3? It also doesn't seem to list all the subdependencies or something-- I noticed it's quite a bit smaller. Not sure if that's something that we need to be concerned about.
> > 
> > 
> > Hmm not really sure I didn't change anything just ran `npm install` and that was what it gave me.
> 
> What nvm/node version do you have?

v16.16.0 node version

File: package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (nturinski) [inline package-lock.json:1]: This looks much better and makes me much more comfortable 😄

**Review by nturinski — APPROVED** [review]

**Pull request merged**